frontend/android/app/build.gradle

```diff
@@ -61,10 +92,20 @@ android {
         }
     }
 
-    // Exclude mapbox-android-accounts because of its proprietary license
     applicationVariants.all { variant ->
-        variant.getCompileConfiguration().exclude group: 'com.mapbox.mapboxsdk', module: 'mapbox-android-accounts'
-        variant.getRuntimeConfiguration().exclude group: 'com.mapbox.mapboxsdk', module: 'mapbox-android-accounts'
+        // If we create a FLOSS build, then we want to exclude GMS
+        if (dartEnvironmentVariables.ANDROID_FLOSS) {
```

Comment:
This is one way to configure builds from the `flutter` cli tool.  Not sure yet how we want to do this.
There are multiple dimensions for creating builds:

* Dimension 1: Release, Debug, Profile build
* Dimension 2: Production, Staging, Local Server
* Dimension 3: Nürnberg, Bavaria, BaWü etc.
* Dimension 4: FLOSS or GAPPS
Any combination of the 4 dimension makes sense:

* Release, Production, Nürnberg, FLOSS
* Debug, Staging, Nürnberg, GAPPS

So we need to come up with a good concept.

Reply:
Yep, but it seems the whitelabeling stuff should be discussed in another issue.
For now, the approach with a env variable seems very fine to me.